deploy/runtime/cashflow.lk

```diff
@@ -1308,12 +1318,15 @@ function set_om(xl) {
 
 	if ( is_fom_fin )
 	{
-		xl_set_output( xl, 'cf_utility_bill' );
+		// no electricity purchases for TPO Host / Developer because system is BTM
+		if ( fin != 'Host Developer' )
```

Comment:
Same comment as above - it may be code is hitting this line due to error on 1319

Reply:
This was bad logic in the set_annual_values() function.